recipe/build-mpi.sh

```diff
@@ -14,6 +14,8 @@ unset FFLAGS F77 F90 F95
 wrapper_ldflags="-L$PREFIX/lib -Wl,-rpath,$PREFIX/lib"
 export LIBRARY_PATH="$PREFIX/lib"
 if [[ "$target_platform" == osx-* ]]; then
+    # FIXME: remove autogen when autotools patch no longer required
+    perl autogen.pl --force
```

Comment:
Then we need the perl dependency in the recipe. Sorry, I removed it a while ago because it was no longer needed.